DOCS-1558 Update docs for "APP-3453 Do not bill for cloud slam compute" (#2328)
viamrobotics/docs: docs/mobility/slam/cartographer/_index.md

{{% alert title="Info" color="info" %}}

> Running `cartographer` in the cloud incurs a small cost per Viam's [Pricing](https://www.viam.com/product/pricing). Currently, cost is only incurred for "Data Management", "Cloud Data Upload", and "Cloud Data Egress".

Reviewer: Let's change it to this to speak more to the user - and yeah we can merge after the app ticket merges
```suggestion
Running `cartographer` in the cloud incurs cost for Data Management, Cloud Data Upload, and Cloud Data Egress. Currently, you incur no cost for compute.
See Viam's [Pricing](https://www.viam.com/product/pricing) for more information. 
```
